spec/values/calculation/abs.hrx

```diff
@@ -64,6 +64,17 @@ a {
   b: 2px;
 }
 
+<===> preserves_single_unit/warning
+DEPRECATION WARNING: In future versions of Sass, abs() will be interpreted as the CSS abs() calculation. This doesn't allow unitless numbers to be mixed with numbers with units. If you want to use the Sass function, call math.abs() instead.
+
+See https://sass-lang.com/d/import
+
+  ,
+1 | a {b: abs(1 + 1px)}
+  |           ^^^^^^^
```

Comment:
Nit: Shouldn't these point to the function name instead?

Reply:
Since the deprecation warning specifically refers to mixing unitless and unit numbers, I think it makes sense to point to the specific location where that mixing occurs.